pkg/syncer/util.go

```diff
@@ -305,8 +305,29 @@ func initVolumeMigrationService(ctx context.Context, metadataSyncer *metadataSyn
 		return nil
 	}
 	var err error
+	var volManager volumes.Manager
+
+	if !metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.MultiVCenterCSITopology) {
+		volManager = metadataSyncer.volumeManager
+	} else {
+		if len(metadataSyncer.configInfo.Cfg.VirtualCenter) > 1 {
+			// Migration feature switch is enabled and multi vCenter feature is enabled, and
+			// Kubernetes Cluster is spread on multiple vCenter Servers.
+			return logger.LogNewErrorf(log,
+				"volume-migration feature is not supported on Multi-vCenter deployment")
+		}
+
+		// It is a single VC setup with Multi VC FSS enabled, we need to pick up the one and only volume manager in inventory.
+		vCenter := metadataSyncer.configInfo.Cfg.Global.VCenterIP
+		cnsVolumeMgr, volMgrFound := metadataSyncer.volumeManagers[vCenter]
```

Comment:
nit: `volManager, volMgrFound := metadataSyncer.volumeManagers[vCenter]`

Reply:
I am setting volManager at line 326.
I cannot use volManager here as it is already declared and is being initialised in different if blocks based on different conditions.